lib/localparticipant.js

```diff
@@ -477,7 +477,7 @@ class LocalParticipant extends Participant {
       throw E.INVALID_TYPE('networkQualityConfiguration', 'NetworkQualityConfiguration');
     }
     ['local', 'remote'].forEach(prop => {
-      if (prop in networkQualityConfiguration && typeof networkQualityConfiguration[prop] !== 'number') {
+      if (prop in networkQualityConfiguration && (typeof networkQualityConfiguration[prop] !== 'number' || isNaN(networkQualityConfiguration[prop]))) {
```

Comment:
Should we create a new function in util and use it here?

Reply:
it does not look like such utility function would have much reuse. Even in this it won't be very useful in other instance where e check for:
```  
    if (type && typeof value !== type) {
      return E.INVALID_TYPE(`${name}.${prop}`, type);
    }
    if (type === 'number' && isNaN(value)) {
      return E.INVALID_TYPE(`${name}.${prop}`, type);
    }
```